src/main/java/edu/oregonstate/cs361/battleship/HardAI.java

```diff
@@ -0,0 +1,113 @@
+package edu.oregonstate.cs361.battleship;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Random;
+
+/**
+ * Created by Will on 3/16/17.
+ */
+public class HardAI extends BattleshipModel {
```

Comment:
I like the thinking here to easily give the AI class access to battleshipmodel members. If we were to have more sprints, I think we should have battleship model have an AI class, easy or hard, rather than having the AI class inherit from battleship model itself